src/main/java/com/prajina/academy/service/CourseServiceImpl.java

```diff
@@ -0,0 +1,142 @@
+package com.prajina.academy.service;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+
+import org.springframework.stereotype.Service;
+
+import com.prajina.academy.model.Course;
+
+@Service("CourseService")
```

Comment:
you don't need to specify "CourseService" string, because you don't have any conflicting implementations yet.